sec/review.tex

```diff
@@ -399,7 +408,8 @@ \subsubsection{Density-based clustering}
 trouble of choosing precise parameters. Examples include the popular OPTICS
 algorithm~\cite{Ankerst1999}, DBCLASD~\cite{Xu1998} for spatial data, and
 DSets-DBSCAN~\cite{Hou2016}. The last provides a clustering that is independent
-of the parameters provided, but is exclusively for the clustering of images. 
+of the parameters provided, but is exclusively for the clustering of images.
+% TODO Some sort of figure to complement the other two?
```

Comment:
Will do. I'll see if there are any decent example datasets elsewhere for showing this. Otherwise, I'll just choose two/three that are well-known.

I think the scikit-learn docs offer some [nice examples](https://scikit-learn.org/stable/auto_examples/cluster/plot_cluster_comparison.html#sphx-glr-auto-examples-cluster-plot-cluster-comparison-py) although I'll stick to the dendogram for hierarchical clustering.


![](https://scikit-learn.org/stable/_images/sphx_glr_plot_cluster_comparison_001.png)

Reply:
In fact, why don't I just create some blobs like these? That's easy enough to do and demonstrates where the algorithms perform best.

Reply:
Yeah: sounds good. What's your thinking for the tree diagram (will scroll down, no doubt you've said something sensible down there).

Reply:
I think a separate figure for the dendogram would be good -- probably just generated with a clustering from "end of section" figures.

I'm thinking each paradigm's subsection closes with a figure showing how a representative method from that paradigm clusters three (seems as good a number as any) examples from the shapes above.

My preference of shapes:
- moons (2nd row): highlights non-linearly separable clusters
- ellipses (4th row): non-spherical, convex, linearly separable clusters, maybe some outliers
- spheres (5th row): spherical and convex, linearly separable, all methods are able to cluster effectively

Reply:
> I'm thinking each paradigm's subsection closes with a figure showing how a representative method from that paradigm clusters three (seems as good a number as any) examples from the shapes above.

Perfect. :+1:

Reply:
Cool. I've opened a PR to this branch on your fork (https://github.com/drvinceknight/literature-review/pull/1) if you could have a quick look. I'm pretty happy with it but the markers might need revising.